nginx/templates/nginx.conf

```diff
@@ -17,17 +16,27 @@
 
 
 user              nginx;
-worker_processes  1;
+# the following has been parametrized via map.jinja in the
+# for loop (see above)
+# worker_processes  1;
```

Comment:
I have removed the comments because they would end up in the pillar which is not right as they are meant for the template. 
I have tried using the if but it doesn't work. Ciprian also advised to follow the existing pattern which is to add the default value in map.jinja.